File: Python3/프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    if(len(arr1) != len(arr2)):
        return -1 if len(arr2) > len(arr1) else 1
    sum1 = sum(arr1)
    sum2 = sum(arr2)
    return -1 if sum2 > sum1 else int(bool(sum1 - sum2))

File: C++/프로그래머스/0/181856. 배열 비교하기/배열 비교하기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> arr1, vector<int> arr2) {
    if(arr1.size() != arr2.size()) {
        return (int)(arr1.size() > arr2.size()) - (int)(arr2.size() > arr1.size());
    }
    
    int sum1 = 0, sum2 = 0;
    for(int i = 0; i < arr1.size(); i++) {
        sum1 += arr1[i];
        sum2 += arr2[i];
    }
    return (int)(sum1 > sum2) - (int)(sum2 > sum1);
}